pypeit/core/load.py

```diff
@@ -336,10 +336,10 @@ def load_1dspec(fname, exten=None, extract='OPT', objname=None, flux=False):
     rsp_kwargs['wave_tag'] = '{:s}_WAVE'.format(extract)
     if flux:
         rsp_kwargs['flux_tag'] = '{:s}_FLAM'.format(extract)
-        rsp_kwargs['ivar_tag'] = '{:s}_FLAM_IVAR'.format(extract)
+        rsp_kwargs['sig_tag'] = '{:s}_FLAM_SIG'.format(extract)
```

Comment:
Not sure why we switched to SIG. 
IVAR works fine.

Reply:
I want to have both. It is annoying to have to take 1/sqrt(ivar). Linetools deals with this, but in general I think we should just write out both SIG and IVAR to disk.